pandas/core/frame.py

```diff
@@ -4829,7 +4829,7 @@ def drop_duplicates(self, subset=None, keep="first", inplace=False):
         duplicated = self.duplicated(subset, keep=keep)
 
         if inplace:
-            inds, = (-duplicated)._ndarray_values.nonzero()
+            (inds,) = (-duplicated)._ndarray_values.nonzero()
```

Comment:
don't change unrelated things

Reply:
sorry black made those changes and I blindly accepted. I am changing my version of black into 19.3b0 and rerunning it.

Reply:
fixed